### PR TITLE
Create 2.2.0 dev image

### PR DESCRIPTION
**Pull request by jedcunningham**
**What this PR does / why we need it**:

Publishes the 2.2.0 dev image so it can be tested.

**Special notes for your reviewer**:

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] If a new distribution or Airflow version is added, please add in .circleci/generate_circleci_config.py and run the script
- [x] If a new distribution or Airflow version is added, there Dockerfile in all base image directories
- [x] If a new Airflow version is added, it supports all distributions

### CODE REVIEW

File: 2.2.0/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+Astronomer Certified 2.2.0-1, 2021-08-24
+----------------------------------------
+
+User-facing CHANGELOG for AC 2.2.0+astro.1 from Airflow 2.2.0:
+
+### Bugfixes
+
+- [astro] Handle Istio containers with Kubernetes Executor Pod adoption (#1318) ([commit](https://github.com/astronomer/airflow/commit/26d52248b))
+- [astro] [AIRFLOW-5448] Handle istio-proxy for Kubernetes Pods (#62) ([commit](https://github.com/astronomer/airflow/commit/dc8b2a999))
```
Comment (jedcunningham) [inline 2.2.0/CHANGELOG.md:11]: With these now being squashed I wasn't sure how we wanted to represent these, so I left them as-is.

File: .circleci/generate_circleci_config.py [hunk 2]
```diff
@@ -21,6 +21,7 @@
     ("2.1.0-3", ["buster"]),
     ("2.1.1-2", ["buster"]),
     ("2.1.3-1", ["buster"]),
+    ("2.2.0-1", ["buster"]),
```
Comment (kaxil) [inline .circleci/generate_circleci_config.py:24]:
```suggestion
    ("2.2.0-1.dev", ["buster"]),
```

Similar to https://github.com/astronomer/ap-airflow/commit/7d1c32cba40f21319e8e7e940ce65319808629cd#diff-801a26920fa2a3670f73ced4973ecb336bb2d11e8afd234edce7d8437a5ec752

Having `.dev` here will add more configs to `.circlecli/config.yml` to have nightly builds. You will have to run this file again after this change

File: 2.2.0/CHANGELOG.md (hunk 1 above)
Comment (kaxil) [reply 2.2.0/CHANGELOG.md:11]: We can squash them here too, general idea is users can look here and visit actual commits in the release branch (v2-2-0 in this case)

File: 2.2.0/buster/build-time-pip-constraints.txt [hunk 3]
```diff
@@ -0,0 +1,527 @@
...
+distlib==0.3.2
+distributed==2.19.0
+dnspython==1.16.0
+docker==5.0.0
+docopt==0.6.2
+docutils==0.16
+ecdsa==0.17.0
+elasticsearch-dbapi==0.2.5
+elasticsearch-dsl==7.4.0
+elasticsearch==7.14.0
```
Comment (kaxil) [inline 2.2.0/buster/build-time-pip-constraints.txt:204]:
```suggestion
elasticsearch==7.13.4
```

**Review by kaxil — APPROVED** [review]

File: 2.2.0/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+Astronomer Certified 2.2.0-1, 2021-08-24
```
Comment (kaxil) [inline 2.2.0/CHANGELOG.md:3]:
```suggestion
Astronomer Certified 2.2.0-1.dev, TBC
```

Since it is not released yet.

**Pull request merged**